internal/providers/azure/uuid.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/banzaicloud/pipeline/internal/global"
 )
 
-const managedByPipelineTag = "io.banzaicloud.pipeline.uuid"
+const managedByPipelineTag = "banzaicloud-pipeline-uuid"
```

Comment:
if now they are the same, Azure could also use the global UUID tag name

Reply:
True dat